package.json

```diff
@@ -216,5 +217,15 @@
       "cloudteam-td-agent-setup",
       "dial-agent"
     ]
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ],
+    "*.css": [
```

Comment:
Simorgh uses css-in-js so we have no `.css` file to lint. Have a look at the `test:lint:css` script we already have and make sure stylelint is run against those same files.

Reply:
Done